Documentation/BuildInstructionsMacOS.md

```diff
@@ -36,6 +36,10 @@ brew install llvm@18
 brew install gcc@13
 ```
 
+# Nix
+
+If you have nix installed you can just enter the devShell provided by the flake and execute the run script.
+
```

Comment:
Providing the command to run would be nice, although I do realize that users might not have enabled the `nix-command` and `flakes` features beforehand.